API.md

```diff
@@ -870,10 +870,14 @@ If there was an error, expect an HTTP status code in either the 4XX or 5XX range
 
 Writes bytes to a data object.
 
+This operation supports two forms of sending data to the HTTP API server. Prefer **form 2**.
```

Comment:
Should we consider Form 1 deprecated? Or is there still a use case for it?

Also, a slight tweak to the wording... Feel free to ignore.
```suggestion
This operation supports two forms of sending data to the HTTP API server. **Form 2** is the recommended form.
```

Reply:
Form 1 is still valid. There might be a use-case for it, but it's difficult to assert that right now. I think we leave it in until we have more information from the community.

Will update wording.

Reply:
Wording updated.

@alanking Leaving open in case you have more questions about form 1.

Reply:
Makes sense to me. No further questions.